packages/tools/client-debugger/client-debugger-view/src/DevtoolsView.tsx

```diff
@@ -0,0 +1,449 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+
+import {
+	ContainerList,
+	ContainerMetadata,
+	DevtoolsFeature,
+	DevtoolsFeatureFlags,
+	DevtoolsFeatures,
+	GetContainerList,
+	GetDevtoolsFeatures,
+	handleIncomingMessage,
+	InboundHandlers,
+	ISourcedDevtoolsMessage,
+} from "@fluid-tools/client-debugger";
+
+import { IStackItemStyles, IStackStyles, Stack } from "@fluentui/react";
+import { FluentProvider } from "@fluentui/react-components";
+import {
+	ContainerDevtoolsView,
+	TelemetryView,
+	MenuItem,
+	MenuSection,
+	LandingView,
+	Waiting,
+} from "./components";
+import { initializeFluentUiIcons } from "./InitializeIcons";
+import { useMessageRelay } from "./MessageRelayContext";
+import { getFluentUIThemeToUse } from "./ThemeHelper";
+
+const loggingContext = "INLINE(DebuggerPanel)";
+
+// Ensure FluentUI icons are initialized.
+initializeFluentUiIcons();
+
+/**
+ * Message sent to the webpage to query for the supported set of Devtools features.
+ */
+const getSupportedFeaturesMessage = GetDevtoolsFeatures.createMessage();
+
+/**
+ * Message sent to the webpage to query for the full container list.
+ */
+const getContainerListMessage = GetContainerList.createMessage();
```

Comment:
Nit: these end up living forever, I think? They're pretty small, so I don't have a huge issue leaving them here, but pointing it out.

Reply:
Yeah, they will. But I figure that's probably a small win over having them regenerated each time the component re-runs.

Reply:
Pretty small cost either way, let's keep them here.